name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/HTMLTableQuoteFeed.java

```diff
@@ -458,9 +458,12 @@ private LatestSecurityPrice extractPrice(Element row, List<Spec> specs) throws P
 
         LatestSecurityPrice price = new LatestSecurityPrice();
 
-        for (Spec spec : specs)
-            spec.column.setValue(cells.get(spec.index), price);
-
+        for (Spec spec : specs){
+            if(!"-".equalsIgnoreCase(cells.get(spec.index).text().trim())){
```

Comment:
Ich bin mir nicht sicher, aber könnte es nicht jetzt auch ein `LatestSecurityPrice` Objekt geben, dass keinen gültigen Werte mehr hat, also z.B. keinen Preis? Das würde dann in der Liste landen. Da ich anfangs prüfe, ob die "Mindest" Specs da sind, kann ich normalerweise von einem gültigen Price Objekt ausgehen (bzw. das eine Exception geworfen wird). Was könnte man machen? Vielleicht das Price Objekt prüfen? Oder man baut die Prüfung auf "-" in die LowColumn bzw. HighColumn ein?